packages/core/test/integration/dialects/postgres/sync-alter-enum.test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const chai = require('chai');
+
+const assert = chai.assert;
+const Support = require('../../support');
+
+const { DataTypes } = require('@sequelize/core');
+
+const dialect = Support.getTestDialect();
+
+if (dialect === 'postgres') {
+  describe('[POSTGRES Specific] sync with alter method with dataType enum', () => {
+    it('properly sync, #7649', async function () {
+      this.sequelize.define('Media', {
+        type: DataTypes.ENUM([
+          'video', 'audio',
+        ]),
+      });
+
+      await this.sequelize
+        .sync({ alter: true })
+        .catch(error => {
+          assert.fail(error.message);
+        });
```

Comment:
this catch is redundant since sync throwing an error will already cause this test to fail (same for the other ones)

Reply:
Fixed